src/components/forms/Checkbox/Checkbox.stories.tsx

```diff
@@ -21,6 +21,11 @@ Basic.args = {
   helpText: 'With onions, mushrooms and green peppers 🍕',
 }
 
+Basic.argTypes = {
+  label: { control: 'text' },
+  helpText: { control: 'text' },
+}
```

Comment:
Since `label` and `helpText` take a `React.ReactNode` type, we need to mark the control as  ̀text` so the user can only enters text. Otherwise, the user could put JSON or pretty much anything, but usage is less straightforward.

Props that don't require an explicit `text` control are those who use `string` as a type and not `React.ReactNode`.

Reply:
For your info, later on, we will maybe have a JSX editor as a control: https://github.com/storybookjs/storybook/issues/11428

Reply:
Got it, thanks! 👍